trailing slash documentation fix

ayezinzu opened the pull request:
Closes #42

ayezinzu committed:
trailing slash documentation fix

alexiglesias93 reviewed (changes requested):
Thank you!
Please:
- Add a changeset
- Reference the #42 issue in the PR

ayezinzu committed:
added changeset

alexiglesias93 commented on package.json:
```diff
@@ -57,5 +57,8 @@
     "typescript": "^4.9.5",
     "vitepress": "1.0.0-alpha.50",
     "vue": "^3.2.47"
+  },
+  "dependencies": {
+    "changeset": "^0.2.6"
```
unrelated dependency, please remove

ayezinzu committed:
removing unused dependency

alexiglesias93 reviewed (changes requested):
CI is failing

ayezinzu committed:
fix lockfile issue

ayezinzu committed:
updated lockfile

ayezinzu committed:
added npmrc to fix lockfile error due to version missmatch

ayezinzu commented:
@alexiglesias93 Issues resolved

alexiglesias93 reviewed (changes requested):
No updates to `.npmrc` are required...
Please get rid of it and the `pnpm-lock.yaml` updates.
Just run `pnpm install`

ayezinzu committed:
removing npmrc and pnpm-lock.yaml updates

ayezinzu commented:
> No updates to `.npmrc` are required... Please get rid of it and the `pnpm-lock.yaml` updates. Just run `pnpm install`

![image](https://github.com/finsweet/ts-utils/assets/59067019/b72c6400-677b-44f9-8783-f7751938e43c)

@alexiglesias93 Doing so causes this error. 

This is what I added the npmrc for

ayezinzu committed:
updated lockfile

ayezinzu commented:
> > No updates to `.npmrc` are required... Please get rid of it and the `pnpm-lock.yaml` updates. Just run `pnpm install`
> 
> ![image](https://private-user-images.githubusercontent.com/59067019/318797444-b72c6400-677b-44f9-8783-f7751938e43c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTIwNjUzNTIsIm5iZiI6MTcxMjA2NTA1MiwicGF0aCI6Ii81OTA2NzAxOS8zMTg3OTc0NDQtYjcyYzY0MDAtNjc3Yi00NGY5LTg3ODMtZjc3NTE5MzhlNDNjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDAyVDEzMzczMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZjYjdkOTJjNmJhNmE4NWUwN2U3MzE4MDlhZTFjNGE1YzRiYTliODllY2IzYWM3ODRlZGNiN2FlNWMyMzVmNDUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ZqvPdGbhI7l3Zd8u-zATxxGXNl-66KCnyUzHF5HNMbU)
> 
> @alexiglesias93 Doing so causes this error.
> 
> This is what I added the npmrc for

Removed npmrc and updated lockfile. 

This fixes the CI error

@alexiglesias93

alexiglesias93 committed:
Update settings.json
